ui/src/tag/AddTagDialog.tsx

```diff
@@ -25,16 +26,19 @@ interface AddTagDialogProps {
 export const AddTagDialog: React.FC<AddTagDialogProps> = ({close, open, initialName, onAdded = () => {}}) => {
     const [name, setName] = React.useState(initialName);
     const [color, setColor] = React.useState('#e6b3b3');
+    const [error, setError] = React.useState('');
 
     const [addTag] = useMutation<AddTag, AddTagVariables>(gqlTags.AddTag, {refetchQueries: [{query: gqlTags.Tags}]});
     const submit = (e: React.FormEvent) => {
         e.preventDefault();
-        addTag({variables: {name, color}}).then((result: MutationFetchResult<AddTag> | void) => {
-            close();
-            if (result && result.data && result.data.createTag) {
-                onAdded(result.data.createTag);
-            }
-        });
+        addTag({variables: {name, color}})
+            .then((result: MutationFetchResult<AddTag> | void) => {
+                close();
+                if (result && result.data && result.data.createTag) {
+                    onAdded(result.data.createTag);
+                }
+            })
+            .catch((err) => setError(err.toString()));
```

Comment:
For consistency with the rest of the UI use the handleError function instead of manually displaying the error.
```typescript
const {enqueueSnackbar} = useSnackbar();
```
```suggestion
            .catch(handleError('Add Tag', enqueueSnackbar));
```